docs/snowflake-native-apps/linkedin-ads.md

```diff
@@ -96,6 +96,12 @@ GRANT USAGE ON SCHEMA <your_schema> TO APPLICATION AIRBYTE_LINKEDIN_ADS;
 GRANT READ ON SECRET integration_linkedin_ads_oauth TO APPLICATION AIRBYTE_LINKEDIN_ADS;
 ```
 
+8. Grant permissions for the app to create a warehouse on which to execute sync tasks, and to execute tasks.
+```
+GRANT CREATE WAREHOUSE ON ACCOUNT TO APPLICATION AIRBYTE_LINKEDIN_ADS;
+GRANT EXECUTE TASK ON ACCOUNT TO APPLICATION AIRBYTE_LINKEDIN_ADS;
```

Comment:
do we also need to include the grant for `MANAGE WAREHOUSES`? Since we also have that permission in the manifest https://github.com/airbytehq/snowflake-native-apps/pull/105/files#diff-6f2bb6ee73e3dc02c5500d9f3387e4478deba90475e8f10f94f1390b81dc4d3eR426-R427

Reply:
We do not, I've removed it in https://github.com/airbytehq/snowflake-native-apps/pull/112.